judge/views/blog.py

```diff
@@ -201,6 +200,23 @@ def get_comment_page(self):
     def get_context_data(self, **kwargs):
         context = super(PostView, self).get_context_data(**kwargs)
         context["og_image"] = self.object.og_image
+        context["valid_user"] = False
+        context["valid_org"] = []
+        for author in self.object.authors.all():
```

Comment:
chỗ này có thể sửa lại đơn giản hơn:
```
if self.request.profile in self.object.authors.all()
```
thay vì for

Reply:
trong đa số TH thì mình chỉ cần dùng profile chứ không cần dùng user (user là model sẵn của django, profile là model tự viết). Một số TH phải dùng user như username, first_name